src/core/pandoc/self-contained.ts

```diff
@@ -33,6 +36,7 @@ export const pandocIngestSelfContainedContent = async (file: string) => {
   cmd.push("--output", filename);
   cmd.push("--metadata", "title=placeholder");
   cmd.push("--embed-resources");
+  resourcePath ? cmd.push("--resource-path", resourcePath.join(":")) : null;
```

Comment:
Our style tends to be to use more explicit statement. Can we just use an if statement here?

Reply:
Oh sure. I'll remember that. Thanks ! 

> Our style tends to be to use more explicit statement. 

We don't have those rule written somewhere right ? By "our style" your are referring to most of current codebase style ? Meaning if I want to know the style to use, it is by observation of the rest of the code ? 

Just curious to improve my next PRs.

Reply:
> Oh sure. I'll remember that. Thanks !
> 
> > Our style tends to be to use more explicit statement.
> 
> We don't have those rule written somewhere right ? By "our style" your are referring to most of current codebase style ? Meaning if I want to know the style to use, it is by observation of the rest of the code ?

Yes, that's what I mean.

But it's also what I think is _correct_ :). Specifically: putting a side-effect in the middle of an expression that you need to know is only conditionally evaluated is asking for trouble. You have `: null` there that isn't doing anything either. Your statement minimizes number of characters, but that's not really a great measure for readability. The replacement is explicit about conditional code paths, and doesn't need a dangling empty `else` clause.

Reply:
Understood. It is clear now. Thank you for the great review and feedback !